mhs/inbound/main.py

```diff
@@ -71,15 +86,12 @@ def start_inbound_server(local_certs_file: str, ca_certs_file: str, key_file: st
                                                              work_description_store=persistence_store,
                                                              config_manager=config_manager))])
 
-    # Ensure Client authentication
-    ssl_ctx = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
-    ssl_ctx.load_cert_chain(local_certs_file, key_file)
-    # The docs suggest we have to specify both that we must verify the client cert and the locations
-    ssl_ctx.verify_mode = ssl.CERT_REQUIRED
-    ssl_ctx.load_verify_locations(ca_certs_file)
+    ssl_ctx = build_ssl_context(local_certs_file, ca_certs_file, key_file) \
+        if str2bool(config.get_config('INBOUND_USE_SSL', default=str(True))) \
```

Comment:
(optional) maybe put this `if` inside `build_ssl_context`

Reply:
I'm not sure about that. `build_ssl_context` is strictly to build it without knowing whether will it be used or not.

Reply:
Yeah, the idea was just to make `start_inbound_server` less complex. I marked that as optional as both ways are fine